projects/cashmere/src/lib/pagination/pagination.component.html

```diff
@@ -1,28 +1,42 @@
-<button hc-button color="secondary" class="hc-page-button hc-left-button" (click)="_previousPage()" [disabled]="!totalPages || _isFirstPage">
-    <hc-icon fontSet="fa" fontIcon="fa-chevron-left"></hc-icon>&nbsp;
-    <span>PREV</span>
-</button>
-<!-- page navigation buttons to appear normally -->
-<button hc-button color="secondary"
-        class="hc-page-button hc-inner-button expanded-page-button"
-        *ngFor="let page of _visiblePages"
-        [disabled]="!page"
-        [class.currentPage]="page === _pageNumber"
-        (click)="_goToPage(page)">
-    <span *ngIf="!!page">{{page}}</span>
-    <hc-icon *ngIf="!page" fontSet="fa" fontIcon="fa-ellipsis-h"></hc-icon>
-</button>
-<!-- page navigation buttons to appear when space is limited -->
-<button hc-button color="secondary"
-        class="hc-page-button hc-inner-button collapsed-page-button"
-        *ngFor="let page of _collapsedVisiblePages"
-        [disabled]="!page"
-        [class.currentPage]="page === _pageNumber"
-        (click)="_goToPage(page)">
-    <span *ngIf="!!page">{{page}}</span>
-    <hc-icon *ngIf="!page" fontSet="fa" fontIcon="fa-ellipsis-h"></hc-icon>
-</button>
-<button hc-button color="secondary" class="hc-page-button hc-right-button" (click)="_nextPage()" [disabled]="!totalPages || _isLastPage">
-    <span>NEXT</span>&nbsp;
-    <hc-icon fontSet="fa" fontIcon="fa-chevron-right"></hc-icon>
-</button>
+<div class="hc-pagination-container">
+    <div *ngIf="!hidePageSize" class="hc-font-sm hc-page-detail-container">
+        <label>Showing:</label>
+        <hc-select [value]="pageSize" (change)="_changePageSize($event.target.value)">
+            <option *ngFor="let pageSizeOption of _displayedPageSizeOptions" [value]="pageSizeOption">
+                {{pageSizeOption}}
+            </option>
+        </hc-select>
+        <label>of <strong>{{length}}</strong> entries</label>
+    </div>
+
+    <div class="hc-page-btns-container">
+        <button hc-button color="secondary" class="hc-page-button hc-left-button" (click)="_previousPage()" [disabled]="!totalPages || _isFirstPage">
```

Comment:
I didn't realize that the last paginator was relying on other cashmere components. I suggest that we keep other cashmere components out of our templates since a subtle change in one could drastically effect the style of another. This can be fixed in a future update

Reply:
But then we are creating similar things that look and act the same way as each other but pull from different code.  So for example, if we change the styling of the button - it comes across here.  If this was something that just looked like the button, we'd have to remember to change its style as well when we updated the button.  And we wouldn't want to rewrite the select box for this component, no?

Reply:
Pagination buttons look nothing like an hc-button. I've been in projects that try to heavily reuse components and extend them where they are used by either changing the default functionality or style. If all of our components start using other components within well eventually run into major issues. By major issues I mean that you can run into cyclic issues. On top of that you have hidden dependencies that the user doesn't know about which could cause issues. Say that you have a drop-down that uses an hc-button and the user the client doesn't import hc-button, then you'll have issues.

Basically there are many reasons not to and few reasons to use cashmere components in other cashmere components templates

Reply:
It doesn't have to be changed in this PR but we should have the understanding that in the future this will need to be changed and that future components should avoid using other cashmere components in their templates